cdap-api-common/src/main/java/io/cdap/cdap/api/exception/ErrorCategory.java

```diff
@@ -71,6 +83,8 @@ public enum ErrorCategoryEnum {
     PLUGIN("Plugin"),
     NETWORKING("Networking"),
     PROVISIONING("Provisioning"),
+    STARTING("Starting"),
```

Comment:
I am just curious, why this order? Looks pretty random

Reply:
yeah, it was random. I sorted the order.